sechub-shared-kernel/src/main/java/com/daimler/sechub/sharedkernel/storage/MultiStorageService.java

```diff
@@ -54,7 +59,12 @@ public MultiStorageService(SharedVolumeSetup sharedVolumeSetup, S3Setup s3Setup)
 
 	@Override
 	public JobStorage getJobStorage(String projectId, UUID jobUUID) {
-		return jobStorageFactory.createJobStorage(projectId, jobUUID);
+	    /* we use here "jobstorage/${projectId} - so we have same job storage path as before in sechub itself
+	     * - for PDS own prefix (storageId) is used ubsude storagePath. We could have changed to
```

Comment:
What is `ubsude`?

Reply:
keyboard finger slide ... it is "inside"... 

changed